### DISCUSSION TITLE
Datepicker: the format to display to the user in the select input is always in english even when the specified locale is "fr"

### DISCUSSION THREAD

**Discussion by ines-gherissi-IPD** [formkit/formkit#1561] [General]
I'm using Formkit to create a datepicker input but the date format to display to the user in the select input is always in english (always DD/MM/YYYY instead of JJ/MM/AAAA) even when the specified locale is "fr". Is there a way to fix that?
![Capture-datepicker](https://github.com/user-attachments/assets/9bd723e3-96c2-4712-917b-70247c474533)

**Comment by justin-schroeder** [comment]
Unfortunately there is not currently a way to change the placeholder tokens. Certainly something we can work on.
